src/lib.rs

```diff
@@ -124,6 +131,38 @@ macro_rules! bitfield_impl {
             $self.0[i] $op $rhs.0[i];
         }
     }};
+    (new for struct $name:ident([$t:ty]); $($rest:tt)*) => {
+        impl<T: AsMut<[$t]> + Default> $name<T> {
+            bitfield_constructor!{T::default(); () -> {}; $($rest)*}
+        }
+    };
+    (new for struct $name:ident($t:ty); $($rest:tt)*) => {
+        impl $name {
+            bitfield_constructor!{0; () -> {}; $($rest)*}
```

Comment:
Is there any reason to use 0 directly instead of default here?

Reply:
I just realized they can all use default; I'll switch that over as soon as my computer cooperates